Build/validate-qir.ps1

```diff
@@ -30,19 +30,33 @@ function Build-One {
         Write-Host "##vso[task.logissue type=error;]Failed to build $Project."
         $script:allOk = $False
     } else {
+        Write-Host "##[info]Compiling $projectName to executable..."
         qir-cli `
             --dll (Join-Path bin $Env:BUILD_CONFIGURATION netcoreapp3.1 "${projectName}.dll") `
             --exe qir
-        if  ($LastExitCode -ne 0) {
-            Write-Host "##vso[task.logissue type=error;]Failed to build $Project."
+        if  ($LastExitCode -ne 0 -or (Get-ChildItem (Join-Path qir *__Interop.exe)).count -eq 0) {
+            Write-Host "##vso[task.logissue type=error;]Failed to compile $projectName to executable."
             $script:allOk = $False
         } else {
+            Write-Host "##[info]Updating environment path variables."
+            $qirPath = Join-Path $projectDirectory qir
+            $old_DYLD_LIBRARY_PATH = $env:DYLD_LIBRARY_PATH
+            $old_LD_LIBRARY_PATH = $env:LD_LIBRARY_PATH
+            $env:DYLD_LIBRARY_PATH += ":" + $qirPath + ":"
```

Comment:
```suggestion
            $Env:DYLD_LIBRARY_PATH += ":${qirPath}:"
```
Note that the `{}` are required here because of the colons following the name of the variable. Without `{}`, `qirPath` is interpreted as a variable scope similar to `Env`, `Script`, `Global`, or `Using`:
![image](https://user-images.githubusercontent.com/31516/124191795-2b764000-da79-11eb-9054-aa66694cca4c.png)

Reply:
I'm not an expert on powershell, but the issue you raise doesn't seem to be the case here. The script worked correctly on Linux, Mac, and Windows.

Reply:
done